Conch/cpanfile.snapshot

```diff
@@ -118,16 +118,18 @@ DISTRIBUTIONS
       ExtUtils::MakeMaker 0
```

Comment:
This cpanfile.snapshot lowers the version on a  lot of modules. Mind getting an update?